tests/Pest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Test Case
+|--------------------------------------------------------------------------
+|
+| The closure you provide to your test functions is always bound to a specific PHPUnit test
+| case class. By default, that class is "PHPUnit\Framework\TestCase". Of course, you may
+| need to change it using the "uses()" function to bind a different classes or traits.
+|
+*/
+
+uses(Illuminate\Foundation\Testing\TestCase::class)->in('Feature', 'Unit');
```

Comment:
There is also talk of unit tests here again, even though they have been removed. But see the comment below.